src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -962,7 +962,29 @@ function getWarningMessage() {
  * @param textAreaId - Id of text area for which char are to be counted
  * @param charCountId - Id of Label to display length of text area
  */
-function updateTextQuestionCharCount(textAreaId, charCountId) {
-    var cs = $('#' + textAreaId).val().length;
-    $('#' + charCountId).text(cs);
+function updateTextQuestionWordsCount(textAreaId, wordsCountId, recommendedLength) {
+	
+    var $response = $('#' + textAreaId).val();
+    var $wordsCountElement = $('#' + wordsCountId);
+    var $wordsCountElementText = $('#' + wordsCountId).text();
```

Comment:
Similarly, not necessary for `$wordscountElementText` to have `$` as it is not a jQuery object